app/questionnaire/placeholder_transforms.py

```diff
@@ -129,21 +129,51 @@ def parse_date(date):
     def add(lhs, rhs):
         return lhs + rhs
 
-    @staticmethod
-    def format_ordinal(number_to_format, determiner=None):
-
-        if 11 <= number_to_format % 100 <= 13:
-            suffix = "th"
-        else:
-            suffix = {1: "st", 2: "nd", 3: "rd"}.get(number_to_format % 10, "th")
-
-        if determiner == "a_or_an":
-            a_or_an = (
-                "an"
-                if str(number_to_format).startswith("8") or number_to_format in [11, 18]
-                else "a"
-            )
-            return f"{a_or_an} {number_to_format}{suffix}"
+    def format_ordinal(self, number_to_format, determiner=None):
+        if self.language in ["en", "eo"]:
+            if 11 <= number_to_format % 100 <= 13:
+                suffix = "th"
+            else:
+                suffix = {1: "st", 2: "nd", 3: "rd"}.get(number_to_format % 10, "th")
+
+            if determiner == "a_or_an":
+                a_or_an = (
+                    "an"
+                    if str(number_to_format).startswith("8")
+                    or number_to_format in [11, 18]
+                    else "a"
+                )
+                return f"{a_or_an} {number_to_format}{suffix}"
+
+        elif self.language == "ga":
+            suffix = "ú"
+
+        elif self.language == "cy":
+            if number_to_format in range(21, 39):
+                suffix = "ain"
+            else:
+                suffix = {
+                    1: "af",
+                    2: "ail",
+                    3: "ydd",
+                    4: "ydd",
+                    5: "ed",
+                    6: "ed",
+                    7: "fed",
+                    8: "fed",
+                    9: "fed",
+                    10: "fed",
+                    11: "fed",
+                    12: "fed",
+                    13: "eg",
+                    14: "eg",
+                    15: "fed",
+                    16: "eg",
+                    17: "eg",
+                    18: "fed",
+                    19: "eg",
+                    20: "fed",
+                }.get(number_to_format, "fed")
```

Comment:
Can we separate getting the suffix into a different method to make this a bit clearer?